inscriptions/ISic001320.xml

```diff
@@ -190,21 +195,23 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI140814">
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#AL">
                 <ab>
-                    <lb n="1"/>
-
-
-                    τύμβον ∙ ὁρᾷ∙ς, παροδεῖτα, <supplied reason="lost">πε</supplied>ρικλειτῆς<lb n="2"/>Ῥοδογούνης ∙ |
-                    ἣν ∙ κτάν∙εν οὐχ ὁσίως ❦<lb n="3"/>λάεσι δεινὸς ∙ ἀνήρ ∙ | κλαῦσε δὲ ∙ καὶ ∙ τάρ-<lb n="4"/>χυσε ∙
-                    Ἀβιάνιος ∙ ἣν ∙ παράκοιτιν ∙ | καὶ<lb n="5"/>βαιὴν ∙ στήλῃ ∙ τήνδ’ ∙ ἀπέδωκε ∙ χάριν. |<lb n="6"/>
-                    ὄνομα ∙ τὸ πρίν ∙ με ∙ πᾶς ἔκλῃζεν #⁵⁶<lb n="7"/> ❦ Ἐπαγαθώ, ❦ |<lb n="8"/> νῦν ∙ δὲ
-                    Ῥοδογούνην ∙ βασιλίδος<lb n="9"/> ❦ τὸ ∙ ἐ∙πώνυμον. ❦
-                    <lb n="10"/>
+                    <lb n="1"/><w>τύμβον</w> <g ref="#interpunct">·</g> <w>ὁρᾷ<g ref="#interpunct">·</g>ς</w> <w>παροδεῖτα</w> <w><supplied reason="lost">πε</supplied>ρικλειτῆς</w>
+                    <lb n="2"/><persName type="attested"><name type="idionym">Ῥοδογούνης</name></persName> <g ref="#interpunct">·</g> <w>ἣν</w> <g ref="#interpunct">·</g> <w>κτάν<g ref="#interpunct">·</g>εν</w> <w>οὐχ</w> <w>ὁσίως</w> <g ref="#ivi-leaf">❦</g>
+                    <lb n="3"/><w>λάεσι</w> <w>δεινὸς</w> <g ref="#interpunct">·</g> <w>ἀνήρ</w> <g ref="#interpunct">·</g> <w>κλαῦσε</w> <w>δὲ</w> <g ref="#interpunct">·</g> <w>καὶ</w> <g ref="#interpunct">·</g> <w>τάρ
+                    <lb n="4" break="no"/>χυσε</w> <g ref="#interpunct">·</g> <persName type="attested"><name type="idionym">Ἀβιάνιος</name></persName> <g ref="#interpunct">·</g> <w>ἣν</w> <g ref="#interpunct">·</g> <w>παράκοιτιν</w> <g ref="#interpunct">·</g> <w>καὶ</w>
+                    <lb n="5"/><w>βαιὴν</w> <g ref="#interpunct">·</g> <w>στήλῃ</w> <g ref="#interpunct">·</g> <w>τήνδ’</w> <g ref="#interpunct">·</g> <w>ἀπέδωκε</w> <g ref="#interpunct">·</g> <w>χάριν</w>
+                    <lb n="6"/><w>ὄνομα</w> <g ref="#interpunct">·</g> <w>τὸ</w> <w>πρίν</w> <g ref="#interpunct">·</g> <w>με</w> <g ref="#interpunct">·</g> <w>πᾶς</w> <w>ἔκλῃζεν</w> <g ref="#ivi-leaf" cert="low">❦</g>
+                    <lb n="7"/><g ref="#ivi-leaf">❦</g> <persName type="attested"><name type="idionym">Ἐπαγαθώ</name></persName> <g ref="#ivi-leaf">❦</g> 
+                    <lb n="8"/><w>νῦν</w> <g ref="#interpunct">·</g> <w>δὲ</w> <persName type="attested"><name type="idionym">Ῥοδογούνην</name> <g ref="#interpunct">·</g> <roleName><w>βασιλίδος</w></roleName></persName>
```

Comment:
I don't think this is part of the personal name (rather saying that she shares a name with the queen?)